x-pack/legacy/plugins/security/public/views/management/api_keys.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { npStart } from 'ui/new_platform';
+import routes from 'ui/routes';
+import { getApiKeysBreadcrumbs } from './breadcrumbs';
+
+if (npStart.plugins.security.__legacyCompat.management) {
```

Comment:
If https://github.com/elastic/kibana/pull/52579 merges before you're unblocked by https://github.com/elastic/kibana/pull/53620, what do you think about using the NP management API instead of relying on the legacy router?

Reply:
Would be happy to! I'm tracking that PR in case it lands before this one - I don't want to keep that legacy stuff unless we absolutely need to.

Reply:
Done.